renv/activate.R

```diff
@@ -0,0 +1,185 @@
+
+local({
```

Comment:
this is all just Jenny Bryan (or whoever) code from `renv::init()` right?

Reply:
yes from kevin Ushey mostly - its the bootstrapping script renv uses to activate itself